doc/how_to/state/url.md

```diff
@@ -12,16 +12,16 @@ When starting a server session Panel will attach a `Location` component which ca
 
 ### Core
 
-* **``pathname``** (string): pathname part of the url, e.g. '/user_guide/Interact.html'.
+* **``pathname``** (string): pathname part of the url, e.g. '/how_to/layout/spacing.html'.
 * **``search``** (string): search part of the url e.g. '?color=blue'.
-* **``hash_``** (string): hash part of the url e.g. '#interact'.
+* **``hash_``** (string): hash part of the url e.g. '#margin-parameter'.
 * **``reload``** (bool): Whether or not to reload the page when the url is updated.
     - For independent apps this should be set to True.
     - For integrated or single page apps this should be set to False.
 
 ### Readonly
 
-* **``href``** (string): The full url, e.g. 'https://panel.holoviz.org/user_guide/Interact.html:80?color=blue#interact'.
+* **``href``** (string): The full url, e.g. 'https://panel.holoviz.org:443/how_to/layout/spacing.html?color=blue#margin-parameter'.
```

Comment:
Yes, port after hostname, although in this case I'd just leave it out

```suggestion
* **``href``** (string): The full url, e.g. 'https://panel.holoviz.org/how_to/layout/spacing.html?color=blue#margin-parameter'.
```

Reply:
right, in the default port case, one would typically leave it out, but we want to demonstrate specifying/accessing the port, so I think leave it in here